### ISSUE TITLE
Alerting rule

### ISSUE THREAD

**Issue by maryam-saeidi** [Kian-Aghaei/moon#1]
## 📝 Summary 

Write an endpoint that gets a URL and a time interval and checks that URL according to the interval. Save the result of execution so that it can be accessed via another API.

Useful information about the execution:
- Time of execution
- Result of execution

## ✅ Acceptance Criteria
- Create endpoints to create a rule
- Save the result of the execution of the rule and create an endpoint to get the list of execution history

**Comment by Kian-Aghaei** [comment]
@maryam-saeidi To clarify:

We will have an endpoint that receives requests similar to:

```
{ 
  url: "https://www.example.com/api/fetch"
  interval: 60
}
```

and then the app persists that and start sending GET request for  that specific url and then stores the result for each (simply success/failure) and stores these individual result in connection with the url entity.

Please let me know if my assumptions here are correct.

**Comment by maryam-saeidi** [comment]
Yes, besides the success/failure, we also need to save time for executing the request. Maybe it would be a good idea to save returned HTTP status code as well just to get an idea of why the request failed. (But it can be in a follow-up ticket too :) )
